cdk/csharp/src/src/Cdk/Cdk.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Is the double `/src/src/` folder a .NET/VisualStudio convention?

Reply:
no actually not, but one usually has a src folder containing the projects and a tests folder along side it (.NET project setup).
The parent src came from the cdk init command. 
I propose we change to:
```
/csharp
  /src
    /Cdk
  /tests
    /Cdk.Tests
  .gitignore
  README.md (depending on above comment)
  cdk.json
  Cdk.sln
```

Reply:
@andrestoll sounds good!